index.js

```diff
@@ -49,8 +49,7 @@ class Toolbar {
         let $toolbar;
 
         let $defaultPlugins = plugins.default.map((Plugin) => { // eslint-disable-line no-unused-vars
-            // Render each plugin with the bound click handler
-            return <Plugin onClick={::this.handlePluginClick} />;
+            return <Plugin onClick={this.handlePluginClick.bind(this)} />;
```

Comment:
@jaredly in this specific case the argument is (confusingly) not an event, but I agree that specifically listing out the arguments is very clear and doesn't look bad.